testutils: remove gocheck

lmb opened the pull request:
Use testing instead of gopkg.in/check.v1. Migrate users of IntegrationCheck to IntegrationTest.

giorio94 commented on pkg/testutils/condition_test.go:
Nit. Should these changes be in a separate commit?

lmb replied:
You mean, split removing checkmate from testutils from removing `IntegrationCheck`?

giorio94 replied:
Yep, exactly (if that makes sense to you)

giorio94 reviewed (approved):
Thanks!

lmb committed:
testutils: remove IntegrationCheck

gocheck tests can now call IntegrationTest, migrate them and
remove IntegrationCheck.

Signed-off-by: Lorenz Bauer <lmb@isovalent.com>

lmb committed:
testutils: use testing instead of gocheck

Remove uses of gocheck in favour of testing.

Signed-off-by: Lorenz Bauer <lmb@isovalent.com>

ti-mo reviewed (approved):
:100: :100: :100:

sayboras reviewed (approved):
LGTM :heavy_check_mark:
